test/unit/instrumentation/fastify/spec-builders.test.js

```diff
@@ -5,87 +5,77 @@
 
 'use strict'
 
-const tap = require('tap')
+const test = require('node:test')
+const assert = require('node:assert')
 const sinon = require('sinon')
 const specs = require('../../../../lib/instrumentation/fastify/spec-builders')
 const WebFrameworkShim = require('../../../../lib/shim/webframework-shim')
 const helper = require('../../../lib/agent_helper')
 
-tap.test('Fastify spec builders', (t) => {
-  let agent
-  let shim
-  t.before(() => {
-    agent = helper.loadMockedAgent()
-    shim = new WebFrameworkShim(agent, 'fastify-unit-test')
-  })
+test('Fastify spec builders', async (t) => {
+  const agent = helper.loadMockedAgent()
+  const shim = new WebFrameworkShim(agent, 'fastify-unit-test')
```

Comment:
Do these get mutated by any subtests?

Reply:
I'm not sure, like the connect tests this was in a before block and just hoisted